README.md

```diff
@@ -1,63 +1,120 @@
+    ```console
+    nextflow run nf-core/scflow -profile test,<docker/singularity/podman/shifter/charliecloud/conda/institute>
     ```
 
-    > Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your Institute. If so, you can simply use `-profile <institute>` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
+    > - Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your Institute. If so, you can simply use `-profile <institute>` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
+    > - If you are using `singularity` then the pipeline will auto-detect this and attempt to download the Singularity images directly as opposed to performing a conversion from Docker images. If you are persistently observing issues downloading Singularity images directly due to timeout or network issues then please use the `--singularity_pull_docker_container` parameter to pull and convert the Docker image instead. Alternatively, it is highly recommended to use the [`nf-core download`](https://nf-co.re/tools/#downloading-pipelines-for-offline-use) command to pre-download all of the required containers before running the pipeline and to set the [`NXF_SINGULARITY_CACHEDIR` or `singularity.cacheDir`](https://www.nextflow.io/docs/latest/singularity.html?#singularity-docker-hub) Nextflow options to be able to store and re-use the images from a central location for future pipeline runs.
+    > - If you are using `conda`, it is highly recommended to use the [`NXF_CONDA_CACHEDIR` or `conda.cacheDir`](https://www.nextflow.io/docs/latest/conda.html) settings to store the environments in a central location for future pipeline runs.
+
+### Analyse your own data
 
-4. Start running your own analysis!
+The **nf-core/scflow** pipeline requires three inputs: (1) a two-column manifest file with paths to gene-cell matrices and a unique sample key; (2) a sample sheet with sample information for each input matrix in the manifest file; and, (3) a parameters configuration file ([see parameter documentation](https://nf-co.re/scflow/dev/parameters)).
 
-    <!-- TODO nf-core: Update the example "typical command" below used to run the pipeline -->
+A complete, automated, scalable, and reproducible case-control analysis can then be performed with a single line of code: -
+
+1. Start running your own analysis!
 
     ```bash
-    nextflow run nf-core/scflow -profile <docker/singularity/podman/conda/institute> --input '*_R{1,2}.fastq.gz' --genome GRCh37
+    nextflow run nf-core/scflow \
+    --manifest Manifest.tsv \
+    --input Samplesheet.tsv \
+    -c scflow_params.config \
```

Comment:
Is this by default necessary or can users specify this using "regular" command line parameters, e.g. `--param 0.1 --param2 0.5` ?

Reply:
It's possible to specify the params in a regular way still, yes.  The main purpose of this was to isolate the analysis-specific parameters (currently ~133 and rising with new features), and provide an easy way to share them (e.g. e-mail, supplemental materials, etc.) and apply the same analysis parameters on a new dataset with copy-paste.